appveyor.yml

```diff
@@ -11,12 +11,15 @@ environment:
 platform:
   - x64
 install:
-  - ps: .\appveyor.ps1
+  - ps: .\appveyor\hardwareinfo.ps1
```

Comment:
+ 1 on the scripts to a folder.

What do you think about creating a generic folder for current build scripts (for now just appveyor related ones), named `build-scripts`,  `ci` or something like that (not `build` as it's most commonly used for output folder for c modules), without the name of the vendor, that way we can also add it to the `.npmignore` file.